crates/node/src/networking/p2p/pea2pea.rs

```diff
@@ -105,6 +109,8 @@ impl P2P {
         instance.enable_writing().await;
         instance.enable_disconnect().await;
 
+        metrics::P2P_PROTOCOL_VERSION.set(instance.protocol_version as i64);
```

Comment:
This will provide us the way to monitor protocol levels on the network. :sunglasses: